markdown/developers/sync.md

```diff
@@ -46,7 +46,7 @@ Sorry about this, but there's no way around it..
 You should not be actively working on the main nf-core repository, so we need to bring these changes to your personal fork.
```

Comment:
Can maybe make this a little softer now:

```
You can either work on the branch created for the template sync to fix the merge conflicts or pull the updates to `TEMPLATE` to your own branch.
Working on your fork is recommended if the merge is not trivial (please make a comment on the automated PR to say that you are working on it though).

```

Reply:
Maybe also restructure / rewrite a little to add a paragraph about how simple merge conflicts can be fixed in the GitHub PR interface?